packages/preview-middleware/test/unit/base/flex.test.ts

```diff
@@ -92,11 +93,56 @@ describe('flex', () => {
         });
 
         test('invalid change', () => {
-            const result = writeChange({}, path, logger);
+            const result = writeChange({} as any, path, logger);
```

Comment:
`any` should not be needed here anymore.

Reply:
removed